gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+
```

Comment:
Good to have  code formatters in the gems like you have here with Rubocop and Simplecov.